### PR TITLE
Use new MiniMeToken in Template

### PR DESCRIPTION
**Pull request by willjgriff**
Integrate new MiniMeToken.

Also make ConvictionVoting request token specifiable.

### CODE REVIEW

File: packages/hardhat/contracts/GardensTemplate.sol [hunk 1]
```diff
@@ -183,22 +185,22 @@ contract GardensTemplate is BaseTemplate, AppIdsXDai {
      */
     function createGardenTxTwo(
         uint256[2] _issuanceSettings,
-        uint64[4] _convictionSettings
+        uint64[4] _convictionSettings,
+        address _convictionVotingRequestToken
```
Comment (fabriziovigevani) [inline packages/hardhat/contracts/GardensTemplate.sol:189]: Let's add the param in the doc string

File: packages/hardhat/contracts/appIds/AppIdsRinkeby.sol [hunk 2]
```diff
@@ -7,6 +7,7 @@ contract AppIdsRinkeby is APMNamehash {
     bytes32 public constant HOOKED_TOKEN_MANAGER_APP_ID = 0x3ccad1fc11d5b14e58c1c53a5138a51f4da8d509831bc505e60bb74d88f8bef5; // wrappable-hooked-token-manager.open.aragonpm.eth
     bytes32 public constant DYNAMIC_ISSUANCE_APP_ID = 0xb4534ca120beef0055b5bd2dbbfe396de7648f1c13a4f7bdfd5a9bd9b40fe824; // dynamic-issuance.open.aragonpm.eth
     bytes32 public constant AGREEMENT_APP_ID = 0x41dd0b999b443a19321f2f34fe8078d1af95a1487b49af4c2ca57fb9e3e5331e; // agreement-1hive.open.aragonpm.eth
-    bytes32 public constant DISPUTABLE_VOTING_APP_ID = 0x39aa9e500efe56efda203714d12c78959ecbf71223162614ab5b56eaba014145;
+//    bytes32 public constant DISPUTABLE_VOTING_APP_ID = 0x39aa9e500efe56efda203714d12c78959ecbf71223162614ab5b56eaba014145; Remove this once we confirm the below one works
+    bytes32 public constant DISPUTABLE_VOTING_APP_ID = 0x705b5084c67966bb8e4640b28bab7a1e51e03d209d84e3a04d2a4f7415f93b34; // disputable-voting.open.aragonpm.eth
```
Comment (fabriziovigevani) [inline packages/hardhat/contracts/appIds/AppIdsRinkeby.sol:11]: Is this a new deployment?

**Review by fabriziovigevani — APPROVED** [review]
Awesome @willjgriff!

File: packages/hardhat/contracts/appIds/AppIdsRinkeby.sol (hunk 2 above)
Comment (willjgriff) [reply packages/hardhat/contracts/appIds/AppIdsRinkeby.sol:11]: Nah, but I presume it was deployed there by Aragon, I was hoping we could make the xDai and Rinekby Id's the same.

Comment (willjgriff) [reply packages/hardhat/contracts/appIds/AppIdsRinkeby.sol:11]: We haven't modified DisputableVoting at all btw.

**Pull request merged**